tools/cdi-containerimage-server/BUILD.bazel

```diff
@@ -0,0 +1,16 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_binary", "go_library")
```

Comment:
not sure how to do this in bazel off the top of my head but we should make sure that the binary has no links to c libraries.  In the POC, I set `CGO_ENABLED=0` on `go build` line.

Reply:
It's sure already done in the previous patch using `pure = "on"` which is equivalent to `CGO_ENABLED=0`.